protos/flyteidl/service/dataproxy.proto

```diff
@@ -136,4 +157,11 @@ service DataProxyService {
     //   description: "Creates a read-only http location that is accessible for tasks at runtime."
     // };
   }
+
+  rpc GetData (GetDataRequest) returns (GetDataResponse) {
+    // Takes an address like flyte://v1/proj/development/execid/n2/0/i and return the actual data
+    option (google.api.http) = {
+      get: "/api/v1/data"
```

Comment:
can we change this
```suggestion
      get: "/api/v1/data?{address}"
```

Reply:
is that how you do it?  i don't think you need this for grpc gateway.  this already works like this.  like what i'm sending in postman is
```
http://localhost:8088/api/v1/data?flyte_url=flyte://v1/flytesnacks/development/f6988c7bdad554a4da7a/n0/d
```